gralloc_gbm.cpp

```diff
@@ -311,13 +313,25 @@ struct gbm_device *gbm_dev_create(void)
 {
 	struct gbm_device *gbm;
 	char path[PROPERTY_VALUE_MAX];
+	int path_len;
 	int fd;
 
-	property_get("gralloc.gbm.device", path, "/dev/dri/renderD128");
-	fd = open(path, O_RDWR | O_CLOEXEC);
-	if (fd < 0) {
-		ALOGE("failed to open %s", path);
-		return NULL;
+	path_len = property_get("gralloc.gbm.device", path, "/dev/dri/card*");
+	/* Search for KMS device with the pattern or open it from property */
+	if (path[path_len - 1] == '*') {
+		fd = open_first_kms_dev(path);
+	} else {
+		fd = open(path, O_RDWR | O_CLOEXEC);
+	}
```

Comment:
Seems a bit cheeky to assume that every glob must end in `*` otherwise it will be treated as a regular path. It shouldn't be too bad to call `open_first_kms_dev` unconditionally?